src/lib/httpFetcher.ts

```
@@ -0,0 +1,37 @@
+      })
+      
+      return {
+        status: response.status,
+        json: async () => response.json(),
+        text: async () => response.text()
+      }
+    } catch (error) {
+      //      if (error instanceof DOMException && error.name === 'AbortError') {
+      //        throw new Error(`Request timeout after ${this.timeout}ms`)
+      //      }
+      throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)